classes/migration/install/NavigationMenusMigration.php

```diff
@@ -72,6 +77,8 @@ public function up(): void
             $table->index(['navigation_menu_item_id'], 'navigation_menu_item_assignments_navigation_menu_item_id');
 
             $table->bigInteger('parent_id')->nullable();
+            $table->foreign('parent_id', 'navigation_menu_item_assignments_parent_id')->references('navigation_menu_item_assignment_id')->on('navigation_menu_item_assignments')->onDelete('cascade');
```

Comment:
Invalid reference, `parent_id` refers in fact to the `navigation_menu_items`, it's needed to fix here and on the migration.